File: 0268-missing-number/0268-missing-number.c
int missingNumber(int* nums, int numsSize){
    //the missing contain all 1+2+3+...+n
    int missing = numsSize*(numsSize+1)/2;
    
    //substract all then what will be left will be the missing
    for(int i = 0; i < numsSize; i++){
        missing -= nums[i];
    }
    return missing;
}

File: 0268-missing-number/0268-missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        N = len(nums)
        #first assign missing 1+2+3+...+n
        missing = N*(N+1)/2
        
        #substract all the values in nums and we will find the missing one
        missing -= sum(nums)
        
        return int(missing)